.github/CODEOWNERS

```
@@ -11,8 +11,8 @@
 
 /BASELINE_NOTES.md
 /.integrated_tests.yaml                               @rrsettgast @CusiniM @cssherman @wrtobin @castelletto1 @jhuang2601 @paveltomin
```

We could also make this:
```suggestion
/.integrated_tests.yaml                               
```